src/components/data/data-utils.js

```diff
@@ -759,19 +759,19 @@ export const SingleEntityEditor = ({ entityType, entityName, attributeName, attr
   const boldish = text => span({ style: { fontWeight: 600 } }, [text])
 
   return h(Modal, {
-    title: 'Modify Attribute',
+    title: 'Edit value',
     onDismiss,
     showButtons: false
   }, [
     consideringDelete ?
       h(Fragment, [
-        'Are you sure you want to delete the attribute ', boldish(attributeName),
+        'Are you sure you want to delete the field ', boldish(attributeName),
```

Comment:
Should this (and below) say "value"? I can see how this is somewhat at the intersection of "column" and "value", but "field" feels closer to "column" to me and I _think_ we're dealing with specific rows here, not the entire column of the data table.

There are a few other places where this might apply. What do you see as the distinction between "field" and "value"?

Reply:
Mostly I was going off the design linked in the [ticket](https://broadworkbench.atlassian.net/browse/SU-99). Though I was thinking of "field" as referring to the cell and "value" as referring to its contents.

That thinking may be influenced by the underlying data model though, which doesn't exactly match the UI. The UI shows a table, but any given row doesn't necessarily have all the attributes represented by the columns in the table. So here, delete doesn't clear the value of the selected attribute, it deletes the attribute from the row entirely.

Reply:
Changed to "value" in https://github.com/DataBiosphere/terra-ui/pull/3017/commits/641f4cdf8b5979039adaf17fd08e5d9f4c4a9cc4.